internal/tui/model.go

```
@@ -0,0 +1,1567 @@
+	}
+	return "s"
+}
+
+func truncate(value string, width int) string {
+	if width <= 0 || lipgloss.Width(value) <= width {
+		return value
+	}
+	return ansi.Truncate(value, width, "…")
+}
+
+func min(left, right int) int {
```

golangci-lint (2.13.2)

[error] function min has same name as predeclared identifier

(predeclared)